### DISCUSSION TITLE
Change name dynamically.

### DISCUSSION THREAD

**Discussion by gimmel** [marcokreeft87/room-card#126] [Q&A]
I want to use the name of an entity to display the power consumption of a device. Can I dynamically change the name to indicate the state of an entity? So instead of the name "TV" for the button icon of my TV, I would like to display "150 W", the current power of my power meter switch.

**Comment by marcokreeft87** [comment]
Have you checked the wiki? https://github.com/marcokreeft87/room-card/wiki/Configuration
There are some examples for you there. Search for 'show_state '

**Reply by gimmel** [reply]
Good hint, but this only shows the direct state of the entity, in my case the state of a switch: on or off. But the icon already shows this state, I want to show the power usage attribute. I didn’t find something like that in the documentation.
